app/uk/gov/hmrc/payetaxcalculatorfrontend/controllers/QuickCalcController.scala

```diff
@@ -47,11 +47,13 @@ class QuickCalcController @Inject() (override val messagesApi: MessagesApi,
       formWithErrors => cache.fetchAndGetEntry.map {
         case Some(aggregate) => BadRequest(quick_calc_form(UserTaxCode.form.withGlobalError(
           Messages("quick_calc.about_tax_code.wrong_tax_code")), aggregate.youHaveToldUsItems))
-        case None => BadRequest(quick_calc_form(formWithErrors, Nil))
+        case None => BadRequest(quick_calc_form(UserTaxCode.form.withGlobalError(
+          Messages("quick_calc.about_tax_code.wrong_tax_code")), Nil))
```

Comment:
if possible, could the global error be defined at form level rather then here, but maybe it can't be done